test/TEST-30-ONCLOCKCHANGE/test.sh

```diff
@@ -29,6 +29,12 @@ test_setup() {
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-resolved.service
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-machined.service
 
+        # extend the watchdog
+        cat >$initdir/etc/systemd/system/systemd-timedated.service.d/watchdog.conf <<EOF
```

Comment:
Does the directory exist already? I guess it must since the test passes ;_)

Reply:
ouch...

Reply:
#12278.